k8s/service-cm.yaml

```diff
@@ -0,0 +1,26 @@
+kind: ConfigMap
```

Comment:
this can be considered incomplete work. If we want to avoid hardcoding the config directly into the image, this needs to be loaded by changing the configuration in `k8s/deploy.yaml`

We can save this file into an issue and put it off to be done later.

Reply:
the biggest thing is that it would avoid changes to `services/eth.json` while also risking it becoming out of sync.